apps/sendsignal.go

```diff
@@ -0,0 +1,61 @@
+package apps
+
+import (
+	"context"
+
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+	"github.com/Scalingo/cli/io"
+	"github.com/Scalingo/go-scalingo/v6"
+	"github.com/Scalingo/go-utils/errors"
+)
+
+func keepUniqueContainersWithNames(containers []scalingo.Container, names []string) map[string]scalingo.Container {
+	containersToKill := map[string]scalingo.Container{}
+
+	for _, name := range names {
+		for _, container := range containers {
+			if container.Label == name {
+				containersToKill[name] = container
+			}
+		}
+		if _, ok := containersToKill[name]; !ok {
+			io.Statusf("The name '%v' did not match any container.\n", name)
```

Comment:
Tell me if you want me to print an error here
```suggestion
			io.Errorf("The name '%v' did not match any container.\n", name)
```

Reply:
Yep definitely :)